src/client/components/equalizer/streamButton/streamButton.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import './streamButton.css';
 
 const StartStopStreamButton = (props) => {
```

Comment:
Try to reuse already existent button component, or create reusable button component, for all of your buttons